packages/desktop-client/src/components/transactions/TransactionList.jsx

```diff
@@ -161,13 +164,19 @@ export function TransactionList({
   });
 
   const onNavigateToSchedule = useCallback(scheduleId => {
-    pushModal('schedule-edit', { id: scheduleId });
+    dispatch(pushModal('schedule-edit', { id: scheduleId }));
+  });
+
+  const onNavigateToFilteredTagView = useCallback(noteTag => {
+    const filterConditions = [
+      { field: 'notes', op: 'contains', value: noteTag, type: 'string' },
+    ];
+    onApplyFilters(filterConditions);
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏currently the click on a tag REPLACES the existing filters. Would it be better to instead `append` a filter?

I can see a use case where I filter by category. And then click on a tag to narrow it down even further. Currently that would remove the category filter.

WDYT?